iothub_client/samples/iothub_client_sample_amqp/iothub_client_sample_amqp.c

```diff
@@ -228,8 +228,9 @@ void iothub_client_sample_amqp_run(void)
     }
 }
 
-int main(void)
+int main(int argc, char* const argv[])
 {
+    if (argc > 1) connectionString = argv[1];
```

Comment:
Hi @JuergenKosel,

for the samples we do not expose getting the arguments from command line.
Let's keep the standard used across the other samples and read it explicitly from the static const char* connectionString variable.